pkg/ui/workspaces/cluster-ui/BUILD.bazel

```diff
@@ -1,119 +1,121 @@
-load("@npm//webpack-cli:index.bzl", webpack = "webpack_cli")
+load("@npm_cluster_ui//webpack-cli:index.bzl", webpack = "webpack_cli")
 load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
-load("@npm//@bazel/typescript:index.bzl", "ts_project")
+load("@npm_cluster_ui//@bazel/typescript:index.bzl", "ts_project")
 load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_test")
-load("@npm//typescript:index.bzl", "tsc_test")
-load("@npm//jest:index.bzl", "jest_test")
+load("@npm_cluster_ui//typescript:index.bzl", "tsc_test")
+load("@npm_cluster_ui//jest:index.bzl", "jest_test")
+load("@npm_cluster_ui//eslint:index.bzl", "eslint_test")
 
 # TODO (koorosh): keeping the list of deps up to date is a candidate to be
```

Comment:
Not really relevant to this PR, but do you know how we might go about auto-generating this stuff so we can keep it up to date? Reviewing these changes is pretty tough and updating them seems equally tough.

Reply:
The best I can think of is using [buildozer](https://github.com/bazelbuild/buildtools/blob/master/buildozer/README.md), making this a list of all the declared direct dependencies in `package.json`, and filtering out the few that shouldn't be there.  At that point we're mixing test dependencies with build dependencies and we may as well just use `@npm_cluster_ui//:node_modules`, since that filegroup (or is it a depset?) is pre-defined by rules_nodejs.

The only other thing we could do is something like "use the [TypeScript Compiler API](https://github.com/Microsoft/TypeScript/wiki/Using-the-Compiler-API) to find `import` statements and exclude imports starting with `src/` or `ccl/` or `.`. It'd get us the runtime dependencies, but we'd still miss all of our build tooling 😬 

There's unfortunately not many good options here.